<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>380</height>
     </rect>
    </property>
    <widget class="QWidget" name="s_login">
     <widget class="QLabel" name="s0_l_title">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>HiveChime Data Access System</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s0_l_username">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="s0_l_password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s0_le_username">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="s0_le_password">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="s0_pb_Login">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QLabel" name="s0_l_status">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s0_pb_NewUser">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="s0_pb_ForgotPassword">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Forgot Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_newUser">
     <widget class="QLabel" name="s1_l_title">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>New User</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_username">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_username">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_password">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_password">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_confirmPassword">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_confirmPassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="s1_pb_CreateUser">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create User</string>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_status">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>181</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_firstName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_firstName">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_lastName">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_lastName">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="s1_l_email">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s1_le_email">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_forgotPassword">
     <widget class="QPushButton" name="s2_pb_reset">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="s2_le_username">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="s2_le_newPassword">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="s2_le_confPassword">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="s2_l_title">
      <property name="geometry">
       <rect>
        <x>233</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s2_l_username">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s2_l_newPassword">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>New Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s2_l_confPassword">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s2_l_status">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_homeScreen">
     <widget class="QLabel" name="s3_l_title">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Home Screen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="s3_pb_viewHive">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Hive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="s3_pb_addHive">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Hive</string>
      </property>
     </widget>
     <widget class="QLabel" name="s3_l_status">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>46</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s3_pb_logout">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_addHive">
     <widget class="QLabel" name="s4_l_title">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Hive</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s4_le_hiveID">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="s4_l_hiveID">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hive ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="s4_l_hiveLocation">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hive Location</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s4_le_hiveLocation">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="s4_l_status">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>141</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>Not set up yet to add to database!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s4_pb_add">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="s4_pb_cancel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>330</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_findHive">
     <widget class="QPushButton" name="s5_pb_selectHive">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Hive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="s5_pb_home">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>320</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>181</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="s5_l_currHive">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Hive:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="s5_l_currentHiveStatus">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_hiveScreen">
     <widget class="QLabel" name="s6_l_title">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Hive Screen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s6_pb_editHive">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Hive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="s6_pb_viewData">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="s6_l_status">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s6_pb_back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_editHive">
     <widget class="QLabel" name="s7_l_title">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Hive</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="s7_l_updateLocation">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Update location:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="s7_le_updateLocation">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="s7_pb_submitChanges">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>310</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit Changes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="s_hiveData">
     <widget class="QLabel" name="s8_l_status">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cue the music, folks</string>
      </property>
     </widget>
     <widget class="QLabel" name="s8_l_title">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="s8_pb_back">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>s0_le_password</tabstop>
  <tabstop>s0_pb_Login</tabstop>
  <tabstop>s0_pb_NewUser</tabstop>
  <tabstop>s0_pb_ForgotPassword</tabstop>
  <tabstop>s1_le_username</tabstop>
  <tabstop>s1_le_password</tabstop>
  <tabstop>s1_le_confirmPassword</tabstop>
  <tabstop>s1_le_firstName</tabstop>
  <tabstop>s1_le_lastName</tabstop>
  <tabstop>s1_le_email</tabstop>
  <tabstop>s1_pb_CreateUser</tabstop>
  <tabstop>s2_le_newPassword</tabstop>
  <tabstop>s2_le_confPassword</tabstop>
  <tabstop>s3_pb_viewHive</tabstop>
  <tabstop>s3_pb_addHive</tabstop>
  <tabstop>s3_pb_logout</tabstop>
  <tabstop>s4_le_hiveID</tabstop>
  <tabstop>s4_le_hiveLocation</tabstop>
  <tabstop>s4_pb_add</tabstop>
  <tabstop>s4_pb_cancel</tabstop>
  <tabstop>s5_pb_selectHive</tabstop>
  <tabstop>s5_pb_home</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>s6_pb_editHive</tabstop>
  <tabstop>s6_pb_viewData</tabstop>
  <tabstop>s6_pb_back</tabstop>
  <tabstop>s7_le_updateLocation</tabstop>
  <tabstop>s7_pb_submitChanges</tabstop>
  <tabstop>s8_pb_back</tabstop>
  <tabstop>s0_le_username</tabstop>
  <tabstop>s2_le_username</tabstop>
  <tabstop>s2_pb_reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
